roles/azuracast/upstream/docker-compose.sample.yml

```diff
@@ -0,0 +1,214 @@
+#
+# AzuraCast Docker Compose Configuration File
+#
+# When updating, you will be prompted to replace this file with a new
+# version; you should do this whenever possible to take advantage of
+# new updates.
+#
+# If you need to customize this file, you can create a new file named:
+#   docker-compose.override.yml
+# with any changes you need to make.
+#
+
+services:
+  web:
+    container_name: azuracast
+    image: "ghcr.io/azuracast/azuracast:${AZURACAST_VERSION:-latest}"
+    # Want to customize the HTTP/S ports? Follow the instructions here:
+    # https://docs.azuracast.com/en/administration/docker#using-non-standard-ports
+    ports:
+      - '${AZURACAST_HTTP_PORT:-80}:80'
+      - '${AZURACAST_HTTPS_PORT:-443}:443'
+      - '${AZURACAST_SFTP_PORT:-2022}:2022'
+      # This default mapping is the outgoing and incoming ports for the first 50 stations.
+      # You can override this port mapping in your own docker-compose.override.yml file.
+      # For instructions, see:
+      # https://docs.azuracast.com/en/administration/docker#expanding-the-station-port-range
+      - '8000:8000'
```

Comment:
What's the purpose all these ~150 lines?

FWIW quite a number of these overlap (conflict?) with other IIAB ports in the 8000-to-8500 range.

Should any adjustments and/or clarifications be made &mdash; either here or at https://github.com/iiab/iiab/wiki/IIAB-Networking#list-of-ports--services ?

Reply:
What part of upstream and sample is unclear to you?

Reply:
Just to clarify, are all 4 files below effectively "documentation only" ?

```
root@box:/opt/iiab/iiab/roles/azuracast# find -name "*sample*"
./templates/azuracast.sample.env
./upstream/sample.env
./upstream/docker-compose.sample.yml
./upstream/azuracast.sample.env
```

Reply:
Documentation, upstream is current, the other is from a year ago see e7c85994d584894c9d40d827d308ae47e236078b